src/CloudLfs.Core/Ref.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Microsoft.MixedReality.CloudLfs
+{
+    public class Ref<T> where T : struct
```

Comment:
where is this used?

Reply:
It will be used in the future, async methods cannot have "ref" parameters so you need to construct a class to pass in a value by ref